src/tools/place_user_tool.rb

```diff
@@ -12,8 +15,19 @@ def onLButtonDown(_flags, x, y, view)
     obj = @mouse_input.snapped_object
     if !obj.nil? && obj.is_a?(Node)
       @hub = obj.hub
-      # TODO: remove this default force value here
-      @hub.is_user_attached ? @hub.remove_user : @hub.attach_user(100)
+      # TODO remove this default force value here
```

Comment:
maybe remove default force value from call and put it into constant at top of class.

Reply:
This is resolved due to new changes reading it in from the filename